src/Traits/RandomValuesTrait.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PNX\DrupalTestUtils\Traits;
+
+use Drupal\Component\Utility\Random;
+use Symfony\Component\Validator\ConstraintViolation;
+
+/**
+ * Additional random value generation methods.
+ */
+trait RandomValuesTrait {
+
+  /**
+   * Generates a random URL.
+   *
+   * @return string
+   *   Random URL.
+   */
+  protected function randomUrl(): string {
+    $random = new Random();
+    return \sprintf('https://%s.com/%s', $random->name(), $random->name());
+  }
+
+  /**
+   * Generates a random email.
+   *
+   * @return string
+   *   Random email.
+   */
+  protected function randomEmail(): string {
+    $random = new Random();
+    return \sprintf('%s@%s.com', $random->name(), $random->name());
+  }
+
+  /**
+   * Generates a random lat lon pair.
+   */
+  protected function randomLatLonPair(): string {
+    $lon = $this->randomPoint(-180, 180);
+    $lat = $this->randomPoint(-84, 84);
+    return \sprintf('POINT(%s %s)', $lon, $lat);
```

Comment:
Nit: floats are not strings
```suggestion
    return \sprintf('POINT(%.05n %.05n)', $lon, $lat);
```